engine/src/comment.cc

```diff
@@ -114,16 +119,20 @@ void comment::delete_host_comments(uint64_t host_id) {
           it->second->get_comment_data().c_str(), it->second->get_persistent(),
           it->second->get_source(), it->second->get_expires(),
           it->second->get_expire_time(), it->first);
-      it = comments.erase(it);
+      comments.erase(it++);
```

Comment:
it = comments.erase(it)

Reply:
this the recommanded way by absl. Their method is a void method.

Reply:
this would not compile